src/Raven.Server/Config/Categories/ReplicationConfiguration.cs

```diff
@@ -6,7 +6,14 @@ namespace Raven.Server.Config.Categories
 {
     public class ReplicationConfiguration : ConfigurationCategory
     {
-        [DefaultValue(600)]
+		[Description("Threshold under which a connection is considered active")]
+		[DefaultValue(30)]
+		[TimeUnit(TimeUnit.Seconds)]
+		[ConfigurationEntry("Raven/Replication/ActiveConnectionTimeoutInSec")]
+		[ConfigurationEntry("Raven/Replication/ActiveConnectionTimeout")]
```

Comment:
I think we need an issue for that, unless we have legacy needs, we want to make sure that we have just one authoritative config value.